pages/docs/data-ops/quickstart.mdx

```diff
@@ -5,43 +5,62 @@ import { Callout, Steps } from "nextra-theme-docs";
 The Hashboard CLI lets you build your Hashboard resources from configuration files, with preview builds and deployments.
 
 <Steps>
-### Install the Hashboard cli
+### Install the Hashboard CLI
 
 ```bash copy
 pip install hashboard-cli
 ```
 
-### Signup and auth
+### Sign up or log in
 
-First signup for an account by clicking 'Get started' on [hashboard.com](https://hashboard.com) or you can signup and authenticate with a single command:
+#### New users
 
-```bash copy filename="signup and auth via cli"
+You can sign up for a new Hashboard account with a single command:
+
+```bash copy
 hb signup
 ```
 
-The `token` command will take you through a web-based auth flow and download your access token to the default token location at ~/.hashboard/hb_access_key.json - you can also specify the token location with the environment variable `HASHBOARD_CREDENTIALS_FILEPATH`.
+Alternatively, click <strong>Get started</strong> on [hashboard.com](https://hashboard.com) to create an account through our website.
+
+#### Existing users
 
-```bash filename="if you signed up on the web" copy
+The `token` command will take you through a web-based authentication flow and download a new access key for you. This will be stored at
+the default filepath `~/.hashboard/hb_access_key.json`.
+
+```bash copy
 hb token
 ```
 
+If you'd like to store this token at a different filepath, you can do so by specifying the `HASHBOARD_CREDENTIALS_FILEPATH` environment variable
```

Comment:
Doesn't seem like we need this in a "quick start".  Quick start should be the shortest path to get to value.  Env variables and advanced configuration can go in the `hb token` docs.

Reply:
Down to move those there!